gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.caching=true
```

Comment:
During deploys/releases, a `gradle.properties` file is decrypted, so this would be overwritten during deploys. I can add this property to the encrypted file, though, if we end up going this route.

Reply:
We no longer use an encrypted `gradle.properties` file, so this is not an issue anymore.